R/db-interface.R

```diff
@@ -1,28 +1,28 @@
-#' Copy a `dm`-object to a `src`/`con`
+#' Copy a `dm`-object to a `src` / `con`
 #'
-#' `cdm_copy_to()` takes a `src`- or `con`-object as a first argument,
-#' and a [`dm`] object as a second. The latter is copied to the former. By default
-#' the key constraints will be set (for now only on MSSQL- and Postgres-DBs).
-#' By default temporary tables will be created.
+#' `cdm_copy_to()` takes a `src`-object or a `con`-object as its first argument
```

Comment:
```suggestion
#' `cdm_copy_to()` takes a [dplyr::src_dbi] object or a [DBI::DBIConnection] object as its first argument
```

Reply:
See my comment above

Reply:
The `[]` create links that point to help topics with more information. The general idea perhaps should be described in a long-form article (vignette).